app/views/users/index.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Users</h1>
-<p>Here is all of users</p>
+<% @users.each do |user| %>
+<div class="user-container d-flex">
+  <div class="user-img">
+    <%= image_tag("user/user_logo.png", width:'100%', height:'100%') %>
+  </div>
+  <div class="user-data d-flex">
+    <h2><a href="/users/<%=user.id%>"><%= user.name %></a></h2>
+    <p>
+      <a href="/users/<%=user.id%>/posts/"> Number of posts: <%= user.posts_counter %></a>
+    </p>
+  </div>
+</div>
+<%end %>
```

Comment:
[OPTIONAL]

- It's a better practice to use like this by making partial instead of using `@objects.each`

```ruby
   # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```